helpers/stripQuerystring.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const SafeString = require('handlebars').SafeString;
+const utils = require('handlebars-utils');
+
+const factory = () => {
+    return function(url) {
+        if (url instanceof SafeString) {
+            url = url.toString();
+        }
+        if (utils.isString(url)) {
```

Comment:
Can we change this to a `NOT` case and throw type error and move the return outside.

Reply:
I'm worried about changing the behavior from the existing helper, since this is not a new helper but replacing one that's already in use - if we have a different failure case that might cause weird behavior in themes?